static/app/utils/discover/types.tsx

```diff
@@ -21,6 +21,14 @@ export enum DiscoverDatasets {
 
 export const TOP_EVENT_MODES: string[] = [DisplayModes.TOP5, DisplayModes.DAILYTOP5];
 
+// The modes that support the interval selector
+export const INTERVAL_MODES: string[] = [
```

Comment:
extra nit: Probably should be called `INTERVAL_DISPLAY_MODES` since it's Display pertaining to the intervals